pkg/filestore/filestore.go

```diff
@@ -26,24 +26,34 @@ var (
 
 type Object struct {
 	Path    string
-	Size    int64
 	Content []byte
 }
```

Comment:
Just have one question, what should we do in case we need the other attributes of this object (its size for instance). I think it would be better to make an object at least has all of its attributes, and the ObjectAttrs is for the special case when we don't need its content.
```go
type Object struct {
	Attrs        ObjectAttrs
	Content  []byte
}
// or
type Object struct {
	Path            string
	Size            int64
	Content      []byte
        UpdatedAt int64
}
```
What do you think? 🤔

Reply:
Nice question. Yes, I think including attributes inside the `Object` struct could be helpful in the future. 
But at this time we are not using them (Attrs inside Object) anywhere and only the `Size` attribute can be extracted from `Get` function (we may need another call to get `UpdatedAt`, etc).
So let me say that in the future, when we find out the need for other attributes of an object as you said, we can do that way or add another dedicated function for it like `GetAttributes`.

Reply:
> we can do that way or add another dedicated function for it like GetAttributes.

I get your point. I'm okay with that approach 👍 Then, how about removing the current type Object? Just checked we only use `obj.Content` and the Path attribute in Object type is redundant to me 🤔 
I mean, we only care about the file content (returned by NewReader) and do read/copy around the blob data. If we only care about the blob, then make `Get` returns ReadCloser of that blob and remove NewReader could be better? 🤔

Reply:
> Just checked we only use obj.Content and the Path attribute in Object type is redundant to me 🤔

> If we only care about the blob, then make Get returns ReadCloser of that blob and remove NewReader could be better?

Nice points. You are totally right. In order to avoid this pull request from making too many changes, let me update it as below:

- Remove `Object` struct
- Change `Get` to return `[]byte` instead of `Object` struct
- Change `NewReader` to `GetReader` and make it to be a function of `Getter` interface

How do you think?

Reply:
It's good to me 👍